packages/components/src/DataList/components/DataListHeaderTile/components/DataListSortingOptions.css

```diff
@@ -0,0 +1,44 @@
+.optionsList {
+  position: absolute;
+  top: 100%;
+  left: 0;
+  max-width: var(--popover--width);
+  box-shadow: var(--shadow-base);
+  margin-top: var(--space-smaller);
+  padding: var(--space-small) 0;
+  border: var(--border-base) solid var(--color-border);
+  border-radius: var(--radius-larger);
```

Comment:
Another ask for [retheme radius](https://github.com/GetJobber/atlantis/blob/7c2af0111b3ee141b35861872f7fdf7f1aca3c57/packages/components/src/Combobox/components/ComboboxOption/ComboboxOption.css#L15-L18) to be specified independently

Reply:
Done here: cd2b3022ed9844e16ec123391e6a68a12dd6380b